### PR TITLE
Send tagged builds to update repository

### PR DESCRIPTION
**Pull request by saifulbkhan**
The changes made in this PR, are mostly around:
1. Making some minor changes to configuration files (update application name, labels shown during the installer, release date , etc.)
2. Adding "update repositories" as required by the Qt Installer Framework to be able to allow users to update their existing installations when new releases are available.

### CODE REVIEW

File: config/config.xml [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
-    <ProductUrl>https://elucidatainc.github.io/ElMaven/</ProductUrl>
+    <ProductUrl>https://resources.elucidata.io/elmaven</ProductUrl>
     <InstallerApplicationIcon>maven</InstallerApplicationIcon>
     <InstallerWindowIcon>window.png</InstallerWindowIcon>
     <ControlScript>control_script.qs</ControlScript>
     <WizardStyle>Modern</WizardStyle>
     <TargetDir>@ApplicationsDir@/El-MAVEN</TargetDir>
+    <RemoteRepositories>
+        <Repository>
+            <Url>https://localhost/repository</Url>
```
Comment (rish9511) [inline config/config.xml:14]: @saifulbkhan 
Why localhost ?

Comment (saifulbkhan) [reply config/config.xml:14]: Oh this is [updated by the python script](https://github.com/ElucidataInc/elmaven-packaging/pull/4/files#diff-ba585ac1b4ca2bdf42f860bad8c5bf33R10) to a valid S3 URL. The script runs before creating archives, installers, etc.

Comment (rish9511) [reply config/config.xml:14]: Oops! Missed it

**Review by rish9511 — APPROVED** [review]
Looks fine
Thanks for taking this up

**Pull request merged**